velox/docs/functions/json.rst

```diff
@@ -2,6 +2,95 @@
 JSON Functions
 ==============
 
+JSON Format
+---------------
```

Comment:
nit: I believe these need to line up with the text, e.g. need to be a few characters shorter here.